Jenkinsfile

```diff
@@ -0,0 +1,73 @@
+library 'magic-butler-catalogue'
+def PROJECT_NAME = 'logdna-agent-v2'
+def RUST_IMAGE_REPO = 'us.gcr.io/logdna-k8s/rust'
+
+pipeline {
+    agent any
+    options {
+        timestamps()
+        ansiColor 'xterm'
+    }
+    triggers {
+        cron(env.BRANCH_NAME ==~ /\d\.\d/ ? 'H H 1,15 * *' : '')
+    }
+    stages {
+        stage('Test') {
+            steps {
+                sh """
+                    make lint RUST_IMAGE_REPO=${RUST_IMAGE_REPO}
```

Comment:
i'm not sure why we dont just make this the default in the makefile and avoid this?  Why is the makefile's image different?

Reply:
This is still a public repository and it's a bit awkward if the default tooling doesn't work. Ideally we could make the rust image publicly available.

Reply:
ah yeah.. where did all of that end up ? i'm with you, would be good to make it public, or find some way to just use the public image

Reply:
Moved to this repo: https://github.com/answerbook/docker-images

__Note__: The rust Dockerfile isn't quite up to date with the needs with the latest changes, I needed to tweak some things. I'm holding off on making another PR for that repo just in case there's more tweaking that needs to be done.